pkgs/games/klavaro/default.nix

```diff
@@ -1,12 +1,12 @@
-{ stdenv, fetchurl, makeWrapper, pkgconfig, intltool, curl, gtk3 }:
+{ stdenv, fetchurl, makeWrapper, pkgconfig, intltool, curl, file, gtk3 }:
```

Comment:
Formatting: best practice is to have each on a separate line, and roughly alphabetize everything after makeWrapper.

i.e.:
```
{ stdenv
, fetchurl
, makeWrapper
, curl
...
}:

Reply:
Just for the record, I don't think we should ask contributors to perform those kind of nitpicks. We have tooling which automates that (e.g. `nixpkgs-fmt`) and I really don't think that a human should care about that kind of details during a review.